src/utils/sign.ts

```diff
@@ -46,6 +46,7 @@ export interface OrderCreation extends UnsignedOrder {
   //  - Signature: EIP-712,ETHSIGN
   //  - Owner address: for PRESIGN
   signature: string // 65 bytes encoded as hex without `0x` prefix. r + s + v from the spec
+  quoteId?: number
```

Comment:
Order creation now takes a new parameter

Reply:
Ah I see.
I'm transforming it to string|undefined here https://github.com/cowprotocol/cowswap/pull/750/files#diff-56184923a93807f92f4953a6963ff08441d32076c7d62d71bf313fb717e8404aL36

But anyway, we won't be passing down the quote id to the appData anymore, only when placing the order

Reply:
But will make it accept `null` just to make it one less conversion step